build/yaml/jobs/windows/binaries.yaml

```diff
@@ -5,6 +5,7 @@
 
 parameters:
   IndexSources: false
+  PublishSymbols: false
```

Comment:
>PublishSymbols: false [](start = 2, length = 21)

Is there a pipeline that sets this to true? I don't see it in the code. If there isn't a pipeline that sets this, then we will never publish symbols.

Reply:
Yes, this is set in the Jobs\binaries.yaml and is added for Signed Release builds

---
In reply to: [333118445](https://github.com/microsoft/CLRInstrumentationEngine/pull/134#discussion_r333118445) [](ancestors = 333118445)